src/lib/SX1280Driver/SX1280.cpp

```diff
@@ -307,21 +308,16 @@ void SX1280Driver::SetPacketParamsFLRC(uint8_t HeaderType,
     buf[3] = (uint8_t)syncWord;
 
     // DS_SX1280-1_V3.2.pdf - 16.4 FLRC Modem: Increased PER in FLRC Packets with Synch Word
-    if ((cr == SX1280_FLRC_CR_1_2) && ((buf[0] == 0x8C && buf[1] == 0x38) || (buf[0] == 0x63 && buf[1] == 0x0E)))
+    if (((cr == SX1280_FLRC_CR_1_2) || (cr == SX1280_FLRC_CR_3_4)) &&
```

Comment:
I don't think 3/4 should be included unless the lower part is added.  We don't use it for now so let's just remove it.

Reply:
I've just included the full code to resolve the FLRC_CR_3_4 errata. Having it all in there is 28 bytes so it is not a big deal to not have to consider it again. If you want me to take it all back out that's fine as well / you're welcome to push that change to speed things along with getting everyone working again.